docs/dxp/latest/en/building-applications/objects/building-solutions-with-objects/using-objects-to-create-kanban-boards.md

```diff
@@ -1,3 +1,251 @@
+   | --- | --- | --- | --- | --- |
+   | Task Title | taskTitle | String | n/a | &#10004; |
+   | Description | description | String | n/a |  |
+   | Task Status | taskStatus | Picklist | Task Status | &#10004; |
+   | Progress | progress | Integer | n/a |  |
+
+   ![Add the Task Title, Description, Task Status, and Progress fields to the Object.](./using-objects-to-create-kanban-boards/images/07.png)
+
+1. Click on the *Details* tab and set the following Entry Display and Scope settings.
+
+   | Field | Value |
+   | --- | --- |
+   | Title Field | Task Title |
+   | Scope | Site |
+   | Panel Category Key | Site Administration > People |
+
+   ![Configure the Object's Entry Display and Scope settings.](./using-objects-to-create-kanban-boards/images/08.png)
+
+1. Click *Publish*.
+
+[Publishing an Object](../creating-and-managing-objects/creating-objects.md#publishing-object-drafts) creates and activates the application with its database columns. This also creates a collection provider for displaying the Object's entries using the Collection Display fragment.
+
+Once Published, you can access the Object and [add task entries](#adding-task-entries) via the Liferay UI or Headless APIs.
+
+## Adding Task Entries
+
+Follow these steps to add task entries to your newly published Object via the Liferay UI:
+
+1. Navigate to the default *Liferay DXP* Site, open the *Site Menu* ( ![Site Menu](../../../images/icon-menu.png) ), and go to *People* &rarr; *Tasks*.
+
+1. Use the *Add* button ( ![Add Button](../../../images/icon-add.png) ) to create task entries.
+
+   Due to limitations in the provided kanban fragment, you'll need at least one task for each Task Status.
+
+   ![Add three tasks to the Object.](./using-objects-to-create-kanban-boards/images/09.png)
+
+After adding task entries, you can [build a Page](#building-a-page-for-displaying-tasks) for displaying them.
+
+## Building a Page for Displaying Tasks
+
+Follow these steps to create a kanban board Page:
+
+1. Navigate to the default *Liferay DXP* Site and create a Content Page named *Tasks*.
+
+1. Add a *Container* fragment to the Page.
+
+1. Add a *Grid* fragment to the container and set the Overflow setting to *Visible*. This ensures the fragment's actions menu is fully visible.
+
+1. Add *Heading* fragments to each grid module for each Task Status: *To Do*, *In Progress*, and *Done*.
+
+1. Add a *Collection Display* fragment to each grid module and select the *Tasks* Collection Provider for each fragment.
+
+1. Filter each Collection Display fragment to display only one Task Status, *todo*, *inprogress*, and *done* respectively.
+
+   ![Filter each Collection Display fragment.](./using-objects-to-create-kanban-boards/images/10.png)
+
+1. Drag and drop the *Kanban Task* fragment onto each Collection Display fragment.
+
+   ![Drag and drop the Kanban Task fragment onto each Collection Display fragment.](./using-objects-to-create-kanban-boards/images/11.png)
```

Comment:
I thought I did something wrong when each of my tasks had fields that were *not mapped*. It would be helpful to mention that DXP tells you the fields are *not mapped* to any object fields yet.